proto/cosmos/gov/v1/tx.proto

```diff
@@ -34,11 +38,9 @@ message MsgSubmitProposal {
   option (gogoproto.stringer)         = false;
   option (gogoproto.goproto_getters)  = false;
 
-  google.protobuf.Any content                       = 1 [(cosmos_proto.accepts_interface) = "Content"];
-  repeated cosmos.base.v1beta1.Coin initial_deposit = 2 [
-    (gogoproto.nullable)     = false,
-    (gogoproto.castrepeated) = "github.com/cosmos/cosmos-sdk/types.Coins"
-  ];
+  repeated google.protobuf.Any messages             = 1;
+  repeated cosmos.base.v1beta1.Coin initial_deposit = 2
+      [(gogoproto.nullable) = false, (gogoproto.castrepeated) = "github.com/cosmos/cosmos-sdk/types.Coins"];
   string proposer = 3 [(cosmos_proto.scalar) = "cosmos.AddressString"];
 }
```

Comment:
Sure, my understanding is that we didn't want title and description to be on-chain

Reply:
> We might also want a bytes metadata field to align with the off-chain JSON approach we've been discussing (which we do need to document somewhere probably in these proto files to start).

I opened this issue here https://github.com/cosmos/cosmos-sdk/issues/10490 which I planned to do in a follow up PR

Reply:
If we want some eventual parity with group proposals then won't that mean eventually removing `title` and `description`

Reply:
Just to reiterate what was discussed on Discord: we've decided to remove `MsgSignal`. Proposals won't have a `title` or `description` field, rather they will have a `metadata` field. This will be done in a follow up PR